[datadog_monitor] Improve require_full_window description (#2125)
DataDog/terraform-provider-datadog: docs/resources/monitor.md

- `renotify_interval` (Number) The number of minutes after the last notification before a monitor will re-notify on the current status. It will only re-notify if it's not resolved.
- `renotify_occurrences` (Number) The number of re-notification messages that should be sent on the current status.
- `renotify_statuses` (Set of String) The types of statuses for which re-notification messages should be sent. Valid values are `alert`, `warn`, `no data`.

> - `require_full_window` (Boolean) A boolean indicating whether this monitor needs a full window of data before it's evaluated. Defaults to `true`. We highly recommend you set this to `false` for sparse metrics, otherwise some evaluations will be skipped.

Reviewer: ```suggestion
- `require_full_window` (Boolean) A boolean indicating whether this monitor needs a full window of data before it's evaluated. Defaults to `true`. Datadog strongly recommends you set this to `false` for sparse metrics, otherwise some evaluations may be skipped.
```
